src/lib.rs

```diff
@@ -44,28 +44,22 @@ pub mod note_bytes;
 
 use note_bytes::NoteBytes;
 
-/// The size of a compact note for Sapling and Orchard Vanilla.
+/// The size of a compact note for Sapling and pre-ZSA Orchard.
```

Comment:
we don't want new term `pre-ZSA Orchard`.
`... for Sapling and Orchard` is fine (x2)